content/posts/weekly/September-2022.md

```diff
@@ -22,3 +22,8 @@ weight: 58
   packit now correctly handles pagination of API results. This resolves an issue that caused
   Fedora 37 not to be pointed to from any alias.
   ([packit#1704](https://github.com/packit/packit/pull/1704))
+
+## Week 37 (September 13th – September 19th)
+
+- Using the method `users_with_write_access` you can generate the set of users that have write access to the project and the method `has_write_access(user)` you can find out if the user has write access to the project. ([ogr#742](https://github.com/packit/ogr/pull/742))
+- Added support for filenames specified in source URL fragments, for example: `https://example.com/foo/1.0/download.cgi#/%{name}-%{version}.tar.gz` ([specfile#100](https://github.com/packit/specfile/pull/100))
```

Comment:
I would not mention these in the weekly blog post -- these are more about OGR and Specfile respectively.

You can provide a general sentence about underlying improvements...

Reply:
I too was wondering about this because I remember making the changes to `move-stable` to use `changelog.py` but only on packit/packit-service but https://github.com/packit/deployment/pull/364 added also specfile and ogr so I thought that there was a consensus to also include those.

Reply:
I think that in blog posts we should really focus only on user-facing changes so ignore the libraries (unless it fixes an important bug for our users). I will remove ogr and specfile from the produced blogpost and drop the line about ogr from this week since it's very specific. As Tomas mentioned, the specfile one may be useful

Reply:
I agree with Tomáš in mentioning the specfile change, it manifested as a bug (though harmless) in packit after all.